testsuite/cluster-test/src/deployment.rs

```diff
@@ -129,12 +130,28 @@ impl DeploymentManager {
             image_digest: Some(hash.clone()),
             image_tag: None,
         }];
-        let response = self
-            .aws
-            .ecr()
-            .batch_get_image(get_request)
-            .sync()
-            .map_err(|e| format_err!("Failed to get image {}: {:?}", hash, e))?;
+        // Retry upto 10 times, waiting 10 sec between retries
+        let response = retry(Fixed::from_millis(10_000).take(10), || {
+            self.aws
+                .ecr()
+                .batch_get_image(get_request.clone())
+                .sync()
+                .map_err(|e| {
+                    println!(
+                        "Failed to get image from repository: {:?}. Retrying...",
```

Comment:
nit: can you use `warn!` instead of `println!`?

Reply:
I needed to add dependency on `common/logger`. Is that ok?

Do I need to initialize `logger` somewhere in cluster-test before using the `warn!()` macro?

Reply:
Updated to not depend on common/logger